### PR TITLE
Fuzzy Search plugin names

### LINKED ISSUE #421: Fuzzy Search plugin names
Just like register/unregister
I wanna take this one :)

### PR DESCRIPTION
**Pull request by tanndlin**
Fixes #421

### CODE REVIEW

File: src/app/handlers/command.handler.ts [hunk 1]
```diff
@@ -15,45 +21,66 @@ export class CommandHandler implements IHandler {
...
+    );
+
+    const msg = await message.channel.send(embed);
+    await msg.react(this._CHECK_EMOTE);
+    await msg.react(this._CANCEL_EMOTE);
+
+    const collector = msg.createReactionCollector(
+      (reaction: MessageReaction, user: User) =>
+        [this._CHECK_EMOTE, this._CANCEL_EMOTE].includes(reaction.emoji.name) &&
+        user.id !== msg.author.id, //Only run if its not the bot putting reacts
```
Comment (c650) [inline src/app/handlers/command.handler.ts:56]: `//<space>`

File: src/app/handlers/command.handler.ts [hunk 2]
```diff
@@ -15,45 +21,66 @@ export class CommandHandler implements IHandler {
...
+      {
+        time: 1000 * 60 * 10,
```
Comment (c650) [inline src/app/handlers/command.handler.ts:58]: `moment.duration`

File: src/app/handlers/command.handler.ts (hunk 1 above)
Comment (tanndlin) [reply src/app/handlers/command.handler.ts:56]: Bruh i swear i've looked for this setting in prettier and cant find it 😂

**Review by c650 — COMMENTED** [review]
- only suggest plugins allowed in the channel/dm
- search plugin aliases too

**Pull request merged**